docs/README.md

```diff
@@ -48,7 +48,12 @@ In layman's terms, BTCPay Server is a self-hosted and automated invoicing system
 
 ### How is it different
 
-BTCPay is code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+BTCPay Server is free, open-source, self-hostable software. 
+BTCPay Server is not a company. To use BTCPay Server, [deploy](./Deployment.md) your own instance, or use a [third-party host](./ThirdPartyHosting.md).
+
+ To explore what BTCPay Server has to offer, head over to the [official demo](https://mainnet.demo.btcpayserver.org/Account/Login).
+
+While using BTCPay Server, there is no third-party between a merchant and a customer. The merchant is always in full control of their funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
```

Comment:
I feel like we should avoid:
- using "no third-party" because it might be confusing with our third-party host terminology.
- using merchant & customer on main intro because of people using the software for donations.

@Zaxounette what do you think about these neutral replacements?

Reply:
Agreed, thanks ! :)